src/main/webapp/WEB-INF/view/activity.jsp

```diff
@@ -19,6 +48,19 @@
     <a href="/activity.jsp">Activity Feed</a>
   </nav>
 
+<div id="feed">
+  <ul>
+    <%
+      for (Conversation convo : conversations) {
+        String title = convo.getTitle();
+    %>
+    <li><strong><%= title %></strong></li>
+    <%
+      }
+    %>
+  </ul>
+</div>
+
   <div id="container">
 
     <% if(request.getAttribute("error") != null){ %>
```

Comment:
??? What is going on here?
is this
<% if(request.getAttribute("error") != null){ %>
supposed to be the last line of the file?

Where is the rest of the file?

Reply:
I've fixed the style issues, but I will further review the guide when I have more time. As far as the "missing" part of the file, I have no idea. The code I committed is complete so once I commit again with the new updates hopefully the entire file will show.